safe-global/safe-wallet-web
# fix: Add feature flag for social login

## What it solves

Part of #2452

## How this PR fixes it

Adds a new feature flag `SOCIAL_LOGIN`. If the feature flag is not enabled on a certain chain the Google Login button in the Header and Welcome page will be hidden and the Social Login option in Onboard will be hidden.

Also removes the redirect to safe creation when logging in on the welcome page.

## How to test it

1. Open Safe on staging CGW
2. Switch to Goerli
3. Observe no Google button visible in the header and welcome page
4. Observe no Social Login button visible in the onboard modal

## Screenshots

<img width="573" alt="Screenshot 2023-11-08 at 10 25 15" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/84134f95-1dc6-4e2d-b3dd-b446ba5f459d">
<img width="407" alt="Screenshot 2023-11-08 at 10 25 23" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/89f0fc4f-0e2e-44c4-a898-4a726ab1e644">
<img width="802" alt="Screenshot 2023-11-08 at 10 25 30" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/663fd0c5-0c00-41a8-9d1e-43921a0df196">


## Checklist
